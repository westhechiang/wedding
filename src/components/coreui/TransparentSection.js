import styled from 'styled-components';
import { theme } from '../../ui';

const TransparentSection = styled.div`
  background: ${theme.color.blueTransparent};
  border-radius: ${theme.borderRadius};
  min-width: 550px;
  width: ${props => props.width || 'auto'};
  padding: ${theme.gutter};
  display: block;
  position: ${props => props.position || 'static'};
`;

export default TransparentSection;
