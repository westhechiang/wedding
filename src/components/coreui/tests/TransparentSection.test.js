import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TransparentSection from '../../../components/coreui/TransparentSection';

describe('<TransparentSection />', () => {
  it('should match its empty snapshot', () => {
    const wrapper = shallow(<TransparentSection />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
