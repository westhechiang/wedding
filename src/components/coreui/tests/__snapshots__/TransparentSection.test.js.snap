// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TransparentSection /> should match its empty snapshot 1`] = `
<div
  className="sc-bdVaJa dMDMzx"
/>
`;
